core/contracts/financial_templates/ExpiringMultiParty.sol

```diff
@@ -6,33 +6,5 @@ import "../FixedPoint.sol";
 
 contract ExpiringMultiParty is Liquidatable {
     using FixedPoint for FixedPoint.Unsigned;
```

Comment:
Do you need this `using` line?

Reply:
I dont think it's needed in the `ExpiringMultiParty.sol`, can remove it.